examples/directml/stable_diffusion/stable_diffusion.py

```diff
@@ -109,13 +130,17 @@ def optimize(model_name: str, unoptimized_model_dir: Path, optimized_model_dir:
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
     parser.add_argument("--optimize", action="store_true", help="Runs the optimization step")
+    parser.add_argument("--optimize_provider", type=str, default="directml", help="EP target for inference")
```

Comment:
(minor naming) `--optimization_target`? `--target`? `--target-backend`? `--target-provider`? I get that "EP" is used in the context of ORT, but I wonder if "target" or "backend" make more sense generally, and the verb "optimize_provider" sounds like it's optimizing a *provider*, rather than optimizing a *model* for a given target backend.

Reply:
I intend to remove this parameter when we have ORT 1.15.1, so I'm not too concerned with the name.